src/main/java/com/sap/sgs/phosphor/fosstars/data/github/HasCompanySupport.java

```diff
@@ -16,25 +16,23 @@
   /**
    * Where the info about open-source projects is stored.
    */
-  private final CompanySupportStorage storage;
+  private final CompanySupportStorage company;
 
   /**
    * Initializes a data provider.
    *
-   * @param where A GitHub organization of user name.
-   * @param name A name of a repository.
    * @param github An interface to the GitHub API.
    * @throws IOException If the info about open-source projects can't be loaded.
    */
-  public HasCompanySupport(String where, String name, GitHub github) throws IOException {
-    super(where, name, github);
-    storage = CompanySupportStorage.load();
+  public HasCompanySupport(GitHub github) throws IOException {
+    super(github);
+    company = CompanySupportStorage.load();
   }
 
   @Override
-  public HasCompanySupport update(ValueSet values) {
+  protected HasCompanySupport doUpdate(GitHubProject project, ValueSet values) {
     logger.info("Figuring out if the project is supported by a company ...");
-    values.update(new BooleanValue(SUPPORTED_BY_COMPANY, storage.supported(url)));
+    values.update(SUPPORTED_BY_COMPANY.value(company.supports(project.url())));
```

Comment:
Not a cacheable value?

Reply:
This data provider doesn't make any network request, it just uses a local storage with info about companies. I don't think it needs to cache values. On the other hand, we have #110 to revise how the data providers use the cache.

Reply:
ok got it